gradio/helpers.py

```diff
@@ -764,7 +764,7 @@ def tqdm(
                 self.iterables.append(new_iterable)
                 callback(self.iterables)
                 return self
-            length = len(iterable) if hasattr(iterable, "__len__") else None  # type: ignore
```

Comment:
Looks good to me @brody715 - can you please format the code with ruff (`bash scripts/format_backend.sh`)? We can merge it in then

Reply:
sure! But I cannot access to my PC now. I will do it later.

Reply:
@freddyaboulton Done